### DISCUSSION TITLE
Loading Screen is not Dark when in Dark Mode

### DISCUSSION THREAD

**Discussion by vaskozl** [jesec/flood#283] [Ideas]
## Environment

`jesec/flood:4.5.2`

## Summary

Even when flood is set to dark mode, the loading screen is quite white and shocks me every once in a while when I load flood at night. The screen I am referring to is the one with the following words:

```
Notifications
Torrent Taxonomy
Data Transfer Rate Details
Data Transfer History
Torrent List
```

It would be great if it was dark when the browser is set to dark mode.

## Expected Behavior

Loading screen to be dark.

## Current Behavior

Loading screen bleaches my eyes.

## Possible Solution

Make the loading screen respect dark mode.

## Steps to Reproduce

1. Wait till your computer switches to dark mode
2. Squint 
3. Open flood

**Comment by jesec** [comment]
It is on the roadmap: https://github.com/jesec/flood/projects/3 .

I am moving this to discussions, as this is a feature request.
